src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
-package core.basesyntax;
+import java.util.Random;
 
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        int index = random.nextInt(colors.length);
+        return colors[index].name();
+    }
```

Comment:
You can directly return the result of the expression without creating a local variable. This will make your code cleaner and more concise.